### PR TITLE
I don't know if this is the right thing to do

### PR DESCRIPTION
**Pull request by fml508**
why the fuck do I want to merge the master branch into the BASE(???) Bin-Debug.

Special Thanks to: 
                            --@thetruekys

### CODE REVIEW

File: README.md
Comment (fml508) [inline README.md:1]: We need to add something to it at the end.

**Review by thetruekys — APPROVED** [review]

File: README.md
Comment (fml508) [reply README.md:1]: ### Last but not least.

**Review by thetruekys — COMMENTED** [review]
Seems good. Keep up the **GOOD** work.

**Review by fml508 — COMMENTED** [review]
All good!

**Review by fml508 — COMMENTED** [review]
No change needed. (as of now at least*)

**Pull request merged**